pageserve/app.py

```diff
@@ -76,36 +76,36 @@ def parsed_posts(posts):
 
 def get_events():
     """Gets all sub-events from events service."""
-    # TODO(cmei4444): integrate with events service to pull event info from
-    # database
-    events = [{'event_id': '1',
-               'name': 'concert 1',
-               'description': 'listen to fun music here!',
-               'author': 'admin',
-               'created_at': '7-9-2019',
-               'event_time': '7-10-2019',
-               },
-              {'event_id': '2',
-               'name': 'concert 2',
-               'description': 'listen to fun music here!',
-               'author': 'admin',
-               'created_at': '7-9-2019',
-               'event_time': '7-12-2019',
-               }]
-    return parsed_events(events)
-
-
-def parsed_events(events):
-    # TODO(cmei4444): implement parsing on events pulled from events service in
-    # a format for web display
-    return events
+    url = os.environ.get("EVENTS_ENDPOINT")
```

Comment:
I tend to like making these 'constants' even when that's not totally a thing in python, but collecting these types of environment variable collection into one place might be nice

Reply:
Would this be a good thing to do using app.config (similarly to app.config["COLLECTION"] in #27)?

Reply:
I think so. You could use `app.config["EVENTS_ENDPOINT"]` or similar.

Reply:
Done in 377b8cf